index.js

```diff
@@ -159,5 +159,10 @@ module.exports = {
                 'classPropertiesAllowed': false
             }
         ]
-    }
+    },
+    'overrides': [{
+        // for files matching this pattern
```

Comment:
- Please remove the comment. 
- Please setup a webstorm to use single quotes & reformat this file.